pkg/flexvol/flexvol.go

```diff
@@ -348,10 +307,15 @@ func retryGetVolumeNameFromMountPath(k8sPath, dockerPath string) (string, error)
 
 func getMountID(path string) (string, error) {
 	log.Tracef("getMountID called with %v\n", path)
+	// Replace backward slash to forward slash.
+	backSlash := "\\\\"
```

Comment:
can this be moved to the windows implementation?

Reply:
The change will not impact the linux since backward slash doesn't exists. The complete fn getMountID() has to be moved to _linux and _winodws which is too much effort. I think we should keep it for the time being. thanks